benchmarker/modules/problems/gemm/args.hpp

```diff
@@ -0,0 +1,48 @@
+// traditional BLAS API:
```

Comment:
I guess it'd be a bit mendokusai, but shouldn't this be separated? `get_matrices.hpp`, `parse_args.h` and `parse_args.cpp`?

Reply:
how does it help if it's separated?

Reply:
Probably just my OCD acting up (don't like template code with non-template/compilable code)... can leave it for now.. nothing serious.